torch/testing/_internal/common_methods_invocations.py

```diff
@@ -61,7 +75,9 @@ def __init__(self,
         self.dtypesIfCUDA = dtypesIfCUDA if dtypesIfCUDA is not None else dtypes
         self.dtypesIfROCM = dtypesIfROCM if dtypesIfROCM is not None else dtypes
 
-        self.op = getattr(torch, self.name)
+        if op is None:
+            assert hasattr(torch, self.name)
```

Comment:
nit: if this is to avoid the getattr below to fail, maybe a comment explaining why this could fail (or a runtime error) would help here.

Reply:
Sure.